modulemd/test-modulemd-yaml.c

```diff
@@ -414,6 +415,55 @@ modulemd_yaml_test_v2_stream (YamlFixture *fixture, gconstpointer user_data)
 }
 
 
+static void
+modulemd_yaml_test_validate_nevra (YamlFixture *fixture,
+                                   gconstpointer user_data)
+{
+  const gchar *good = "nodejs-devel-1:8.10.0-3.module_1572+d7ec111e.x86_64";
+  const gchar *garbage = "DEADBEEF";
+  const gchar *garbage2 = "DEAD.BEEF";
+  const gchar *garbage3 = "MORE-DEAD.BEEF";
+  const gchar *missing_epoch =
```

Comment:
why is it important? Epoch is entirely optional and `rpm` and `dnf` do not output it by default in UI.

Reply:
We're being very specific in the format we expect here to simplify parsing expectations. Pungi always includes the 0:, so we're going to validate that if only to ensure consistency when third-parties start producing content.

Reply:
See https://github.com/fedora-modularity/libmodulemd/issues/46

Consumers of libmodulemd want a simpler string to parse. Pungi already guarantees that the `0:` is included, so this is mostly just insurance for future third-parties to make sure they're producing the same content.